examples/ecmascript/es5.ohm

```diff
@@ -338,18 +328,13 @@ ES5 {
                   | "!"    UnaryExpression  -- lnot
                   | PostfixExpression
 
-  MultiplicativeExpression = MultiplicativeExpression "*" UnaryExpression -- mul
-                           | MultiplicativeExpression "/" UnaryExpression -- div
-                           | MultiplicativeExpression "%" UnaryExpression -- mod
+  MultiplicativeExpression = MultiplicativeExpression ("*"|"/"|"%") UnaryExpression -- mul
                            | UnaryExpression
 
-  AdditiveExpression = AdditiveExpression "+" MultiplicativeExpression -- add
-		     | AdditiveExpression "-" MultiplicativeExpression -- sub
-		     | MultiplicativeExpression
+  AdditiveExpression = AdditiveExpression ("+"|"-") MultiplicativeExpression -- add
```

Comment:
That's what I used to call these inline rules, but I was never super happy
with it. How about we just go with "rec" and make a mental note to revisit
this naming convention?

On Fri, Jan 15, 2016 at 12:38 PM, Patrick Dubroy <notifications@github.com>
wrote:

> In examples/ecmascript/es5.ohm
> <https://github.com/cdglabs/ohm/pull/50#discussion_r49902266>:
>
> >                             | UnaryExpression
> >
> > -  AdditiveExpression = AdditiveExpression "+" MultiplicativeExpression -- add
> > -		     | AdditiveExpression "-" MultiplicativeExpression -- sub
> > -		     | MultiplicativeExpression
> > +  AdditiveExpression = AdditiveExpression ("+"|"-") MultiplicativeExpression -- add
>
> With cases like this, I've never known what to use for the inline rule
> name. AdditiveExpression_add just seems weird. But it just occurred to me
> -- what about "rec"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cdglabs/ohm/pull/50/files#r49902266>.
>